utils/ci/run_test.sh

```diff
@@ -1,21 +1,23 @@
 #!/usr/bin/env bash
 set -e
 
-export LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-"$HOME/.local/share/lunarvim"}"
-export LUNARVIM_BASE_DIR="${LUNARVIM_BASE_DIR:-"$LUNARVIM_RUNTIME_DIR/lvim"}"
+export NVIM_APPNAME="${NVIM_APPNAME:-"lvim"}"
+
+export LUNARVIM_DATA_DIR="${LUNARVIM_DATA_DIR:-"$HOME/.local/share/$NVIM_APPNAME"}"
+export LUNARVIM_BASE_DIR="$LUNARVIM_DATA_DIR/core"
 
 export LVIM_TEST_ENV=true
 
 # we should start with an empty configuration
-LUNARVIM_CONFIG_DIR="$(mktemp -d)"
-LUNARVIM_CACHE_DIR="$(mktemp -d)"
+XDG_CACHE_HOME="$(mktemp -d)"
+XDG_CONFIG_HOME="$(mktemp -d)"
 
-export LUNARVIM_CONFIG_DIR LUNARVIM_CACHE_DIR
+export XDG_CACHE_HOME XDG_CONFIG_HOME
 
-printf "cache_dir: %s\nconfig_dir: %s\n" "$LUNARVIM_CACHE_DIR" "$LUNARVIM_CONFIG_DIR"
+printf "cache_dir: %s\nconfig_dir: %s\n" "$XDG_CACHE_HOME" "$XDG_CONFIG_HOME"
 
 lvim() {
-  nvim -u "$LUNARVIM_BASE_DIR/tests/minimal_init.lua" --cmd "set runtimepath+=$LUNARVIM_BASE_DIR" "$@"
+  nvim -u "$LUNARVIM_BASE_DIR/tests/minimal_init.lua" "$@"
```

Comment:
This will completely break other tools that otherwise works correctly now. For example, [ripgrep expects its config file under `XDG_CONFIG_DIR/ripgrep/ripgreprc`](https://github.com/BurntSushi/ripgrep/blob/master/GUIDE.md#configuration-file).